content/docs/user-guide/packaging/project-export-ios.md

```diff
@@ -0,0 +1,84 @@
+---
+linkTitle: Project Export iOS
+title: Project Export for iOS
+description: (Experimental) Learn how to use the Project Export CLI to create an Xcode project to build and deploy for iOS.
+toc: true
+weight: 450
+---
+In this article, we will go over the process to use project export tooling as part of the process to building and deploying games to the iOS platform. First, we will use the iOS export script to generate the necessary Xcode project file and prepare assets. Afterwards we will walk through how to ensure Xcode is setup properly to build an app for iOS.
```

Comment:
This is not an article, this is a user guide. Articles are more appropriate for something like blogs.

Reply:
Or does this entire update belong in an O3DE blog instead of the user guide?